### ISSUE TITLE
[Feature] Option to disable camera roll for character movement

### ISSUE THREAD

**Issue by 0xDE57** [Teardown-Issue-Tracker-Maintainers/Teardown-Issue-Tracker#896]
### Is your feature request related to a problem (if not, explain why you think this feature should be added)? Please describe.

The camera roll while the character is moving is making me nauseous. Particularity when moving side to side and the entire world tilts due to the camera roll. 

I was only able to play for about 5 minutes before having to stop. I have dug through all the options and found no setting. I wasn't even able to finish the first mission / tutorial.

A lot of games have options to disable "view bobbing" such as minecraft. This is more or a rolling than a bobbing, but it is bad enough to the point I might need to refund as this rolling makes it unplayable for me. 

### Describe the solution you'd like

Add a setting to disable camera roll for the first person character camera. Or perhaps a slider so I can make it much less roll. 

### Describe alternatives you've considered

Without a way to disable it, a refund is the only alternative because I cannot play it.

**Comment by Pacman1105** [comment]
This already exists. It's the setting with the 0-100 bar, however I don't remember the name

**Comment by YuLun-bili** [comment]
oh yea indeed @0xDE57 

<img height="540" src="https://github.com/user-attachments/assets/ee5cfcd3-b7a6-4bbe-83c3-28f0a482b32f" />

**Comment by Pacman1105** [comment]
Better issue would be how the viewroll affects aiming (at least for Kooshing's gun mod) and it makes it incredibly hard to use certain mods

**Comment by 0xDE57** [comment]
Oh... Indeed it does. I did not see for some reason yesterday, I must have glossed right over it.

This is now playable at 0. Thank you.
